### PR TITLE
Add methodology sub-nav and empty page for vitals

### PR DESCRIPTION
**Pull request by daschi**
## Description of the change

This PR is part of adding the Vitals methodology, and adds the top right sub-nav on the methodology page and a path that can work for either vitals or projections. Since @jovergaag is working on the design polish for the projections methodology, I'm going to wait to fully implement the vitals methodology until that's ready so we can reuse some of the design components. 

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Part of #982 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/views.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const CORE_PATHS: Record<string, string> = {
   communityVitals: "/community/vitals/:entityId?",
   facilitiesExplore: "/facilities/explore",
   facilitiesProjections: "/facilities/projections",
-  methodology: "/methodology",
+  methodology: "/methodology/:dashboard",
```
Comment (jovergaag) [inline src/core/views.ts:31]: This is not resolving correctly. When clicking on the Methodology link from the Vitals page, the path resolves to `http://localhost:3000/methodology/:dashboard/vitals` and I get a Not Found page.

File: src/core/VitalsMethodology/index.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2020 Recidiviz, Inc.
```
Comment (jovergaag) [inline src/core/VitalsMethodology/index.tsx:2]: 2021

**Review by jovergaag — CHANGES_REQUESTED** [review]
One small path bug.

File: src/core/MethodologyLink.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import * as styles from "./CoreConstants.scss";
+
+const MethodologyLink: React.FC<{ path: string }> = ({ path }) => {
+  return (
+    <Link className="MethodologyLink" to={path}>
+      <Icon
+        className="MethodologyLink__icon"
+        kind={IconSVG.Open}
+        width={12}
+        height={12}
```
Comment (Catacola) [inline src/core/MethodologyLink.tsx:32]: Can this be set in the scss file?

**Comment by Catacola** [comment]
I'm still getting a 404 when I click on the methodology link in vitals:
Screen Shot 2021-04-13 at 10 16 53 AM

The path seems to be correct. Is this expected behavior for this in-between state?

**Comment by daschi** [comment]
@Catacola Oh yeah, it only works if `showMethodologyDropdown` flag is true because that's what's adding the `methodology` navigation options.

**Review by jovergaag — APPROVED** [review]

**Pull request merged**